### ISSUE TITLE
How to use this?

### ISSUE THREAD

**Issue by WouterrV** [mmathias01/google-charts#23]
I'm a newb in the js ecosystem - how do actually use this module in my project? The quick start says to use: `import {GoogleCharts} from 'google-charts';`, but I think that only works when the js-file that I'm putting the example code in is in the same dir as the google-charts package, right? What am I missing?

**Comment by aftab470** [comment]
You just need to install this package first using npm cmd and follow the documentation.

**Comment by WouterrV** [comment]
@aftab470 I did install it using npm cmd. But I still don't understand what's happening. Also the documentation on this project is not exactly extensive.

**Comment by TahinDani** [comment]
Yeah, proper documentation would be handy. I'm also struggling with how to use it in React.
